packages/swirl-components/src/components/swirl-chip/swirl-chip.css

```diff
@@ -54,11 +75,23 @@
     & ::part(avatar) {
       background-color: var(--s-surface-raised-pressed);
     }
+
+    & + .chip__remove-button {
+      background-color: var(--s-surface-overlay-pressed);
+    }
+  }
+}
+
+.chip--icon-color-highlight {
+  & .chip__icon {
+    color: var(--s-icon-highlight);
```

Comment:
**issue (blocking):** seems that this style does not work. i also don't see this class correclty rendered in the browser styles
<img width="603" alt="image" src="https://github.com/getflip/swirl/assets/23700195/f640025a-6968-4c05-9103-2145a609a5d5">

Reply:
The icon color only affects the icon passed via the icon prop.

Reply:
ah ok. my bad then :D 
works!